in-viewport.js

```diff
@@ -53,22 +53,27 @@ function debounce(func, wait, immediate) {
 }
 
 // https://github.com/jquery/sizzle/blob/3136f48b90e3edc84cbaaa6f6f7734ef03775a07/sizzle.js#L708
-var contains = global.document.documentElement.compareDocumentPosition ?
-  function (a, b) {
-    return !!(a.compareDocumentPosition(b) & 16);
-  } :
-  global.document.documentElement.contains ?
-    function (a, b) {
-      return a !== b && ( a.contains ? a.contains(b) : false );
-    } :
-    function (a, b) {
-      while (b = b.parentNode) {
-        if (b === a) {
-          return true;
-        }
-      }
-      return false;
-    };
+var contains = function() {
+  if (global.document) {
+    return global.document.documentElement.compareDocumentPosition ?
+            function (a, b) {
```

Comment:
You should keep the 2 space indentation :rabbit: 
Maybe we should add a `.editorconfig` file?